apps/showcase/app/select.tsx

```diff
@@ -32,7 +32,7 @@ export default function SelectScreen() {
   return (
     <>
       <View className='flex-1 justify-center items-center p-6 gap-12'>
-        <Select>
+        <Select defaultOpen defaultValue={{ value: 'apple', label: 'apple' }}>
```

Comment:
Can you also remove:
_Line 15-21_
```tsx
const VALUES = {
  apple: 'Apple',
  banana: 'Banana',
  blueberry: 'Blueberry',
  grapes: 'Grapes',
  pineapple: 'Pineapple',
};
```

Since it is no longer needed